DaanKort/DataFrame
# Added scripts to run backend and frontend at the same time

## Added new scripts

    "client-install" : "npm install --prefix client", //cds into client and performs an npm install
    "start": "node server.ts", //Runs the frontend
    "server": "nodemon server.ts", //Runs the backend
    "client": "npm start --prefix client", //cds into client folder and runs npm start
    "dev": "concurrently \"npm run server\" \"npm run client\"" //Runs both backend and frontend at the same time
 

### Setup
- Run `npm install` in the root folder
- Run `npm client-install` in the root folder
- Run `npm run dev` to run backend and frontend at the same time

### How to test
- Open a rest client like Postman or Insomnia
- Make a POST request to `http://localhost:8080/api/user`,
- Enter firstName, lastName, password and email and check if you can sign up
- Check if you get any error feedback when leaving something blank
